apps/manager_app/auth_card.c

```diff
@@ -539,6 +581,9 @@ void card_auth_handler(manager_query_t *query) {
     }
 
     send_auth_card_response(&resp);
+    if (MANAGER_AUTH_CARD_RESPONSE_COMMON_ERROR_TAG == resp.which_response) {
```

Comment:
Please consider re-visiting this flow again (when time permits) to improve handling of exit conditions...

Reply:
Agreed, I understand that sending response and errors from here is not ideal, so we can update this as follows
- Each handler should send their own response to host for success case 
- Send error response to host in errors cases before showing error display.
- The parent should only be concerned with correct exit condition i.e. flow complete or error occurred
- Flow complete should be indicated by flow status.
- For early exits, say auth failed after the serial sign, handler can return MANAGER_TASK_FAILED to exit.

Reply:
can add these points as TODO here

Reply:
will do.

Reply:
Got it, please add as to do so we can move ahead